packages/block-library/src/comments-pagination-numbers/block.json

```
@@ -0,0 +1,16 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "core/comments-pagination-numbers",
+	"title": "Comments Pagination Numbers",
+	"category": "design",
+	"parent": [ "core/comments-query-loop" ],
+	"description": "Displays a list of page numbers for comment pagination",
```

Nitpick: the description should the plural form when referring comments similar to the block title:

```suggestion
	"description": "Displays a list of page numbers for comments pagination.",
```